core/templates/dev/head/base.js

```diff
@@ -77,14 +77,14 @@ oppia.constant('MAX_NODE_LABEL_LENGTH', 15);
 // Global utility methods.
 oppia.controller('Base', [
   '$scope', '$http', '$rootScope', '$window', '$timeout', '$document', '$log',
-  'warningsData', 'activeInputData', 'LABEL_FOR_CLEARING_FOCUS',
+  'warningsData', 'LABEL_FOR_CLEARING_FOCUS',
   function(
       $scope, $http, $rootScope, $window, $timeout, $document, $log,
-      warningsData, activeInputData, LABEL_FOR_CLEARING_FOCUS) {
+      warningsData, LABEL_FOR_CLEARING_FOCUS) {
     $rootScope.DEV_MODE = GLOBALS.DEV_MODE;
 
     $scope.warningsData = warningsData;
-    $scope.activeInputData = activeInputData;
+    $scope.activeInputName = '';
```

Comment:
Is this actually needed here? I suspect not, since it's only used in ExplorationSettings.js and you've declared it there already.